sdk/identity/azure-identity/README.md

```diff
@@ -120,16 +120,61 @@ A credential is a class which contains or can obtain the data needed for a servi
 
 The credential types in Azure Identity differ in the types of AAD identities they can authenticate and how they are configured:
 
-|credential class|identity|configuration
-|-|-|-
-|`DefaultAzureCredential`|service principal or managed identity|none for managed identity; [environment variables](#environment-variables) for service principal
-|`ManagedIdentityCredential`|managed identity|`ManagedIdentityCredentialBuilder`
-|`EnvironmentCredential`|service principal|[environment variables](#environment-variables)
-|`ClientSecretCredential`|service principal|`ClientSecretCredentialBuilder`
-|`ClientCertificateCredential`|service principal|`ClientCertificateCredentialBuilder`
-|`DeviceCodeCredential`|user account|`DeviceCodeCredentialBuilder`
-|`InteractiveBrowserCredential`|user account|`InteractiveBrowserCredentialBuilder`
-|`UsernamePasswordCredential`|user account|`UsernamePasswordCredentialBuilder`
+<html>
+<head>
+<table style="border-collapse: collapse;">
+<thead>
+<tr>
+<th style="height:40px; width:200px; border:1px solid #dddddd;">credential class</th>
```

Comment:
Please use css via a style block.

Reply:
I  tried this but I am not able to use <style> tag inside Markdown correctly. It is rendering <style> block as is, like normal text. Can you please let me know if I am doing something wrong?

Reference- 
https://stackoverflow.com/questions/20598628/do-style-tags-work-in-markdown

Reply:
You can just remove all of the CSS and just use the html border property.

Reply:
Thanks for the help. Done as suggested.